modules/Material/Toolbar.qml

```diff
@@ -153,7 +153,7 @@ View {
 
     Row {
         id: windowControls
-
+        visible: clientSideDecorations
```

Comment:
I mentioned it in the pull request. If we are going to be picky and force a separate pull request / branch for a tiny tiny fix like this, than so be it. That being said, I didn't think its a big deal to have this fixed here.